conf/config-default.yaml

```
@@ -111,6 +111,7 @@ apisix:
                                   #   more details.
     ssl: radixtree_sni          # radixtree_sni: match route by SNI(base on radixtree)
   #stream_proxy:                  # TCP/UDP proxy
+  #  enabled: false               # enable stream proxy mode
```

```suggestion
  #  enabled: true                # enable stream proxy mode
```
Will it be better to use `true` ?